lms/templates/main.html

```diff
@@ -165,7 +165,9 @@
 </html>
 
 <%def name="login_query()">${
-  u"?next={0}".format(urlquote_plus(login_redirect_url)) if login_redirect_url else ""
+  u"?next={next}".format(
```

Comment:
I considered adding a test but it would have to be a Bok Choy test and I couldn't find a natural place to add this.